dev-ruby/rgl/rgl-0.6.5.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+USE_RUBY="ruby31"
+RUBY_FAKEGEM_EXTRADOC="README.md"
+
+inherit ruby-fakegem
+
+DESCRIPTION="A Graph Library for Ruby"
+HOMEPAGE="https://github.com/monora/rgl"
+SRC_URI="https://github.com/monora/rgl/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Ruby"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+ruby_add_rdepend "
+	>=dev-ruby/stream-0.5.3
```

Comment:
This dependency does not match the gemspec, which lists "~> 0.5.3". In gentoo terms that matches "=0.5*", but given that we are adding stream as a new package the preference would be to set SLOT=$(ver_cut 1-2) in the stream ebuild and depend on >=dev-ruby/stream-0.5.3:0.5 here.